flow/Knobs.cpp

```diff
@@ -174,6 +174,7 @@ void FlowKnobs::initialize(bool randomize, bool isSimulated) {
 	init( TRACE_RETRY_OPEN_INTERVAL,						  1.00 );
 	init( MIN_TRACE_SEVERITY,                 isSimulated ? 1 : 10 ); // Related to the trace severity in Trace.h
 	init( MAX_TRACE_SUPPRESSIONS,                              1e4 );
+	init( TRACE_DATETIME_ENABLED,                             true ); // trace time in human readable format (always real time)
```

Comment:
shall we make the default false?
We should test both true and false value for this variable using BUGGIFY.

Reply:
I think there was general agreement that defaulting to true was reasonable, but if you think that you aren't going to want this we could discuss more.

Reply:
Testing both together makes sense to me!  I think the result from the last conversation was that we wouldn't mind making this on by default, but we could discuss again

Reply:
I'm fine with what was concluded from the last conversation.